docs/samples/shards/UI/Area/2.shs

```diff
@@ -0,0 +1,71 @@
+@mesh(main)
```

Comment:
This one still contains the manual run boilerplate (probably doesn't work with run-sample)

Reply:
Thanks for catching, forgot to remove after testing. The manual run boilerplate has been removed in [06e0b24](https://github.com/fragcolor-xyz/shards/pull/771/commits/06e0b24f886f2a7cda3d248a783784f61818c0b3)